src/Plugin/views/field/OgMembershipBulkForm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\views\field;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\OgMembershipInterface;
+use Drupal\system\Plugin\views\field\BulkForm;
+
+/**
+ * Defines a user operations bulk form element.
+ *
+ * @ViewsField("og_membership_bulk_form")
+ */
+class OgMembershipBulkForm extends BulkForm {
+
+  /**
+   * {@inheritdoc}
+   *
+   * Provide a more useful title to improve the accessibility.
+   */
+  public function viewsForm(&$form, FormStateInterface $form_state) {
+    // Remove all actions related to (non-)member role.
+    foreach ($this->actions as $key => $action) {
+      if (strstr($key, '-member') !== FALSE) {
```

Comment:
Non-member and member are more 'implied' roles, right?
(\Drupal\og\Entity\OgMembership::getRoles)
Removing the member should remove the actual membership entity, removing the role won't do the trick I guess...
Having those actions (remove membership) is valuable, but I should probably get rid of the  two actions related to the member and non member entirely, and replace it with an action that removes the OgMembership entity itself. But as you're working on those :)
What do you think about removing the role related actions here (not creating them in the insert hook)?